specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2023-04-01-preview/agfood.json

```diff
@@ -22313,10 +22329,19 @@
         "properties": {
           "description": "A collection of key value pairs that belongs to the resource.\r\nEach pair must not have a key greater than 50 characters\r\nand must not have a value greater than 150 characters.\r\nNote: A maximum of 25 key value pairs can be provided for a resource and only string,\r\nnumeral and datetime (yyyy-MM-ddTHH:mm:ssZ) values are supported.",
           "type": "object",
-          "additionalProperties": {},
-          "example": {
-            "key1": "value1",
-            "key2": 123.45
+          "additionalProperties": {}
+        }
+      },
+      "example": {
```

Comment:
this example is not a complete example considering other required fields. Shall we remove from here given `examples` folder have sample request JSON for each API ?